mlir/include/mlir-c/Dialect/SparseTensor.h

```diff
@@ -25,7 +25,9 @@ MLIR_DECLARE_CAPI_DIALECT_REGISTRATION(SparseTensor, sparse_tensor);
 /// These correspond to SparseTensorEncodingAttr::LevelType in the C++ API.
 /// If updating, keep them in sync and update the static_assert in the impl
 /// file.
-enum MlirSparseTensorLevelType {
+typedef uint64_t MlirSparseTensorLevelType;
+
+enum MlirBaseLevelType {
```

Comment:
Since this is not used in too many places, let's stay consistent and use 

MlirSparseTensorLevelBaseType;

or

MlirBaseSparseTensorLevelType;